monix-eval/shared/src/test/scala/monix/eval/TaskLocalSuite.scala

```diff
@@ -19,10 +19,11 @@ package monix.eval
 
 import minitest.SimpleTestSuite
 import monix.execution.Scheduler
+import monix.execution.schedulers.TracingScheduler
```

Comment:
Having users import TracingScheduler by themselves might not be OK.

It’s usage should probably be automatic when that option is on.

Reply:
@alexandru Nice idea. So if the option is on, basically wrap the current scheduler into the `TracingScheduler`. Will work on it.

Reply:
BTW I would like to have `.tracing` method on a scheduler that returns a `TracingScheduler`. For those that are already tracing, it could simply return `this`.

Reply:
@oleg-py that could be an extension method, will try it out.